installconfig.adoc

```diff
@@ -441,7 +441,7 @@ default is **TRUE**.
 
 | *Enable version update notifications.*
 a| This enables automatic notification when updates are available
-when this is set to TRUE. + 
+when this is set to TRUE. +
```

Comment:
nit: for consistency it seems like we should have **TRUE** (bold) - but this doc, like so many others is riddled with inconsistencies :(